src/js/modules/infragistics.ui.widget.js

```diff
@@ -98,36 +99,93 @@
 				return $.ig.regional.defaults[ key ];
 			}
 		},
+		_getLocaleWidgetName: function () {
+			this.localeWidgetName = this.localeWidgetName ||
+							(this.widgetFullName || this.widgetName)
+								.replace(/^((ui-ig(\.)*)|(ig(\.)*))/ig, "");
+			return this.localeWidgetName;
+		},
+		_getLocaleObject: function (locale) {
+			return $.ig.locale &&
```

Comment:
Yep, but the idea was if locale is string - my version of this widget is implemented for the case when this.options.locale is string. I have in mind that it could be an object so I could easily change it.